buildspec.yaml

```diff
@@ -12,7 +12,12 @@ phases:
       - /bin/bash ./build.sh
   post_build:
     commands:
-      - "[ -f .BUILD_COMPLETED ] && [ -z \"${SKIP_PACK:-}\" ] && /bin/bash ./pack.sh"
```

Comment:
Why not follow @rix0rrr's suggestion and implement this inside `pack.sh` itself? Much safer and you can even test it before merging...

Reply:
It feels _wrong_ to me that calling `pack.sh` would not... pack... if some environment variable is set. I rather NOT call `pack.sh` in that case.